### PR TITLE
Unit interface updated to discussed spec for Issue

### PR DESCRIPTION
**Pull request by itwatts**
Changed formatting to be consistent

### CODE REVIEW

**Comment by thewells1024** [comment]
This references ticket "https://github.com/thewells1024/TowerOffense/issues/40"

File: Unit.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+import java.lang.*;
+import java.util.*;
+
+public interface Unit {
```
Comment (thewells1024) [inline Unit.java]: this should also have `public Location getLocation()`

File: Unit.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+import java.lang.*;
+import java.util.List;
+
+public interface Unit {
```
Comment (thewells1024) [inline Unit.java]: this should also have a method for getting the location

File: Unit.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (thewells1024) [inline Unit.java]: remove this line please

File: Unit.java [hunk 4]
```diff
@@ -1,8 +1,7 @@
-
 import java.util.List;
 
 public interface Unit {
```
Comment (thewells1024) [inline Unit.java]: I just realized this should have `public boolean takeDamage(int)`

File: Unit.java [hunk 5]
```diff
@@ -1,8 +1,7 @@
-
 import java.util.List;
 
 public interface Unit {
-	public Location move(List<Location>);
-	public Location getLocation();
-	public void damage(Base);
+   public Location move(List<Location>);
+   public Location getLocation();
+   public void damage(Base);
```
Comment (thewells1024) [inline Unit.java]: Now that I'm thinking about it, this shouldn't be here, but there should be a getter for strength (aka the amount of damage that a unit will do to a base)

File: Unit.java (hunk 3 above)
Comment (primitiveDataType) [reply Unit.java]: We may need some more functions: 
+ Unit(Location): Unit
+ move(List<Location>): Location
+ damage(Base): void
+ getHealth(): double

Units will have attributes:
+ hp: int
+ attack: int
+ speed: int
+ health: double
- texture: image
- location: Location

(Getting this info from the draft class diagram that's in the drive folder).

Comment (thewells1024) [reply Unit.java]: I don't think we want a getHp or getSpeed
Constructors shouldn't be in an interface

**Pull request merged**